src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -877,7 +878,7 @@ export default {
         case FilingTypes.INCORPORATION_APPLICATION:
           // redirect to Edit web app to correct this Incorporation Application
           const editUrl = sessionStorage.getItem('EDIT_URL')
-          const url = `${editUrl}correction/?filingId=${item.filingId}`
+          const url = `${editUrl}${this.entityIncNo}/correction?corrected-id=${item.filingId}`
```

Comment:
Tested locally and I was redirected to:
`https://business-create-dev.pathfinder.gov.bc.ca/businesses/edit/BC1218875/correction/?corrected-id=85818`